docs/requirements.txt

```
@@ -1,7 +1,7 @@
 matplotlib
 numpy
 sphinx-copybutton>=0.3.1
-sphinx-gallery>=0.9.0
+sphinx-gallery >=0.9.0, !=0.11.0
```

Why not just straight out `>= 0.11.1`?

---

Soft BC? If you have `0.10.1` in your env currently, you don't need to update. But, yeah, these are just dev dependencies, so I'm also fine with just going for the full upgrade. Your call.

---

I'm just wondering if the rendering might fail with `< 0.11` considering our new CSS? Since I can't be bothered to check I'd go for the more restrictive `>= 0.11.1` :[

---

Let's go for the upgrade

```suggestion
sphinx-gallery>=0.11.1
```